superset/charts/schemas.py

```diff
@@ -817,6 +900,19 @@ class ChartDataQueryObjectSchema(Schema):
         "as `having_druid`.",
         deprecated=True,
     )
+    druid_time_origin = fields.String(
+        description="Starting point for time grain counting on legacy Druid "
+        "datasources. Used to change e.g. Monday/Sunday first-day-of-week. "
+        "This field is deprecated and should be passed to `extras` "
+        "as `druid_time_origin`.",
+        allow_none=True,
+    )
+    url_params = fields.Dict(
+        description="Optional query parameters passed to a dashboard or Explore view",
+        keys=fields.String(description="The query parameter"),
+        values=fields.String(description="The value of the query parameter"),
+        allow_none=True,
+    )
```

Comment:
Bycatch: this was missing, will be added to `superset-ui/core`, after which url params can be used in v1 chart data requests with jinja templates.